sympy/integrals/transforms.py

```diff
@@ -1870,6 +1938,19 @@ def laplace_transform(f, t, s, legacy_matrix=True, **hints):
     If the integral cannot be fully computed in closed form, this function
     returns an unevaluated :class:`LaplaceTransform` object.
 
+    This function can also transform differentials; in that case the result
+    will contain unevaluated expressions like ``LaplaceTransform(x(t), t, s)``
+    and initial conditions like ``x(0)`` or ``Subs(Derivative(x(t), t), t, 0)``.
```

Comment:
It isn't really "unevaluated expressions". If the function `x(t)` is unknown then what more can we do but refer to `L{x(t)}`. That's also how it displays in LaTeX form:
```python
In [112]: print(latex(laplace_transform(f(t), t, z)))
\mathcal{L}_{t}\left[f{\left(t \right)}\right]\left(z\right)
```
Id you use this in a jupyter notebook then that's the output you'll see (the pretty printer could be changed to shbow the same in a terminal.

Also rather than `Subs(Derivative(x(t), t), t, 0)` I'd just say `x'(0)`. In pretty or latex printing it shows up better e.g.:
```
In [114]: laplace_transform(f(t).diff(t, 2), t, z)
Out[114]: 
 2                                         ⎛d       ⎞│   
z ⋅LaplaceTransform(f(t), t, z) - z⋅f(0) - ⎜──(f(t))⎟│   
                                           ⎝dt      ⎠│t=0
```
(It's really just `LaplaceTransform` that doesn't display very well here)

Reply:
Next version is clearer ;)